db/migrate/20191128003907_add_clearance_to_users.rb

```diff
@@ -0,0 +1,26 @@
+class AddClearanceToUsers < ActiveRecord::Migration[6.0]
+  def self.up
+    change_table :users do |t|
+      t.string :confirmation_token, limit: 128
+      t.string :remember_token, limit: 128
+    end
+
+    add_index :users, :remember_token
+
+    users = select_all("SELECT id FROM users WHERE remember_token IS NULL")
+
+    users.each do |user|
+      update <<-SQL
+        UPDATE users
+        SET remember_token = '#{Clearance::Token.new}'
+        WHERE id = '#{user['id']}'
```

Comment:
There is a school of thought that avoids changing data in migrations. 

Here's a [good primer](https://thoughtbot.com/blog/data-migrations-in-rails). I don't think you should change this one, but I'd feel remiss if I didn't mention the other approach.

Reply:
This is great, thanks! Again, this came from the generator, so I think I'll keep this for now, but bookmarked this for the future.